DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -96,17 +96,23 @@
 DQMOfflinePhysics = cms.Sequence( dqmPhysics )
 
 
+from Validation.Geometry.trackingRecoMaterialAnalyzer_cfi import materialDumperAnalyzer
```

Comment:
@rovere - would prefer not to have this dependency on validation here - can you move things around to keep DQM independent of Validation (which hopefully it is..). Thanks